database/migrations/2020_02_09_201925_create_app_answers_table.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateAppAnswersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('app_answers', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->bigInteger('app_id')->unsigned();
+            $table->bigInteger('question_id')->unsigned();
+            $table->string('answer', 3000);
```

Comment:
the 3000 won't actually do anything in this application

Reply:
seems like it will tho